runtime/autoload/netrw.vim

```diff
@@ -9518,7 +9534,7 @@ fun! s:NetrwTreeDisplay(dir,depth)
     NetrwKeepj call s:NetrwTreeDisplay(direntry.'/',depth)
    elseif entry =~ '@$' && has_key(w:netrw_treedict,direntry.'@')
 "    call Decho("<".direntry."/> is a key in treedict - display subtree for it",'~'.expand("<slnum>"))
-    NetrwKeepj call s:NetrwTreeDisplay(direntry.'/',depth)
+    NetrwKeepj call s:NetrwTreeDisplay(direntry.'@',depth)
```

Comment:
why the change of `/` to `@`?

Reply:
When `has_key(w:netrw_treedict,direntry.'@')`, we must call `s:NetrwTreeDisplay` with the correct name of the key which is `direntry.'@'`

Reply:
so the use of `/` was an error?

Reply:
I think yes. But actually this case probably never happens. As links are usually not listed with trailing `@` in `w:netrw_treedict`. But maybe there are cases where it can happen that I'm not aware of. I think in this case this was an error.